go.mod

```diff
@@ -116,3 +117,5 @@ require (
 
 // See: https://github.com/ghodss/yaml/pull/65
 replace github.com/ghodss/yaml => github.com/sourcegraph/yaml v1.0.1-0.20200714132230-56936252f152
+
+replace github.com/sourcegraph/sourcegraph/lib v0.0.0-20220715132627-cb906941e8b4 => ../sourcegraph/lib
```

Comment:
This will go away when `sourcegraph` PR is merged. This is here for my local development.